app/src/main/java/org/fossasia/susi/ai/MainActivity.java

```diff
@@ -119,7 +120,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     protected void onResume() {
         super.onResume();
-        etMessage.requestFocus();
+        //etMessage.requestFocus();
```

Comment:
I see what happened, I wanted the keyboard to open while opening the app, while the OP didn't. so the question is do we need keyboard to open automatically on startup or not?